File: src/main/java/org/bsplines/ltex_ls/parsing/plaintext/PlaintextAnnotatedTextBuilder.java
package org.bsplines.ltex_ls.parsing.plaintext;

import org.bsplines.ltex_ls.parsing.CodeAnnotatedTextBuilder;

public class PlaintextAnnotatedTextBuilder extends CodeAnnotatedTextBuilder {
  @Override
  public CodeAnnotatedTextBuilder addCode(String code) {
    addText(code);
    return this;
  }
}

File: src/main/kotlin/org/bsplines/ltexls/parsing/plaintext/PlaintextAnnotatedTextBuilder.kt
/* Copyright (C) 2020 Julian Valentin, LTeX Development Community
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/.
 */

package org.bsplines.ltexls.parsing.plaintext

import org.bsplines.ltexls.parsing.CodeAnnotatedTextBuilder

class PlaintextAnnotatedTextBuilder(
  codeLanguageId: String,
) : CodeAnnotatedTextBuilder(codeLanguageId) {
  override fun addCode(code: String): CodeAnnotatedTextBuilder {
    addText(code)
    return this
  }
}
